### ISSUE TITLE
ice

### ISSUE THREAD

**Issue by crashstash** [ice531/hello-world#3]
from pt?

**Comment by ice531** [comment]
Yes. That you crash?

If so...wow....its been a lonnnngggg time my friend.

On Jul 23, 2017 10:24 PM, "crashstash" <notifications@github.com> wrote:

from pt?

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/ice531/hello-world/issues/3>, or mute the thread

**Comment by crashstash** [comment]
YES! I am so glad I was able to track you down! crshstsh@gmail.com

**Comment by ice531** [comment]
Hilarious..i tried finding you awhile back but no luck. Tried emailing your
old ip_net account and even looked on Paltalk related forum's. Did find
that locohacker guy lol

On Jul 23, 2017 10:52 PM, "crashstash" <notifications@github.com> wrote:

YES! I am so glad I was able to track you down! crshstsh@gmail.com

—
You are receiving this because you commented.

Reply to this email directly, view it on GitHub
<https://github.com/ice531/hello-world/issues/3#issuecomment-317306794>,
or mute
the thread

**Comment by crashstash** [comment]
Yeah man I tried a few times before now. Left you offline msg on aim with my contact info and even dated my message like I'm leaving a recording for future generations. 
Tried getting history of domain name you owned and everything. Apparently somebody in Germany goes by the same handle. 

I'd like to catch up and see how things have been with you so I left you my email address. We can go further then if you want to message me. 

Hope to hear from you dude
